2_process/src/process_data.R

```diff
@@ -0,0 +1,31 @@
+# Load Libraries
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+# Create function to process data
+process_data <- function(input_data = '1_fetch/out/model_RMSEs.csv', output_path = '2_process/out/model_summary_results.csv') {
```

Comment:
I think you didn't check in `2_process/out` folder into git (and the file that would be generated here).

Reply:
Hmm. Since the folders are currently empty (I haven't run the entire script yet, which would populate the folders with the .csvs, .png, and .txt files) I think they aren't pushing.  Would you suggest that I run the script ahead of time (in which case I might need a little help on a screenshare!) or that I make placeholder files so that the folders aren't empty?

Reply:
I often create an empty file for that purpose. 

from git bash, 

`touch 2_process/out/empty` 
then
`git add 2_process/out/empty`
`git commit -m 'adding empty file so we track the folder'`
would do it. Does that help?